crates/bevy_scene/src/bundle.rs

```diff
@@ -0,0 +1,96 @@
+use std::ops::{Deref, DerefMut};
+
+use bevy_asset::Handle;
+use bevy_ecs::{
+    bundle::Bundle,
+    change_detection::ResMut,
+    entity::Entity,
+    prelude::{Changed, Component},
+    system::{Commands, Query},
+};
+use bevy_transform::components::{GlobalTransform, Transform};
+
+use crate::{DynamicScene, InstanceId, Scene, SceneSpawner};
+
+/// [`InstanceId`] of a spawned scene. It can be used with the [`SceneSpawner`] to
+/// interact with the spawned scene.
+#[derive(Component)]
+pub struct SceneInstance(InstanceId);
+
+impl Deref for SceneInstance {
+    type Target = InstanceId;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for SceneInstance {
+    fn deref_mut(&mut self) -> &mut <Self as Deref>::Target {
+        &mut self.0
+    }
+}
+
+/// A component bundle for a [`Scene`] root.
+///
+/// Once the scene is spawned, the entity will have a [`SceneInstance`] commponent.
+#[derive(Default, Bundle)]
+pub struct SceneBundle {
```

Comment:
Say I want to add the `Scene` to some entity while keeping the existing transform. I could just do a `commands.entity(to_update).insert(scene_handle)` right? It might be worthwhile adding that info to the `Scene` doc. Maybe here too, but I don't think it is as necessary.

Reply:
Yes, but that's the same for all bundles: if you want to add one to an existing entity without overriding the existing value of component, you can't add the bundle but need to add only the new components

Reply:
Agreed; this should be a seperate bit of functionality that's universal for all bundles.

Reply:
added some basic doc 👍